src/config.rs

```diff
@@ -29,8 +29,8 @@ impl Default for Config {
         Config {
             source: "./".to_owned(),
             dest: "./".to_owned(),
-            layouts: "_layouts".to_owned(),
-            drafts: "_drafts".to_owned(),
+            templates: "templates".to_owned(),
+            drafts: "drafts".to_owned(),
```

Comment:
Ah, I see that you are correct - [`/src/cobalt.rs L23-L25`](https://github.com/cobalt-org/cobalt.rs/blob/master/src/cobalt.rs#L23-L25). I'll add an underscore to `templates`, but -- off-topic -- I wonder if this is really the best way to handle this? Using a pattern for ignoring could cause problems for the user; usually things like this are handled via an explicit list or from a configuration value, like the `ignored` property in `.cobalt.yml`. What are your thoughts? This conversation should probably be in its own issue.

`_drafts` actually shouldn't be changed here, as this PR is strictly for `_layouts -> templates`. I'll remove this change.